### ISSUE TITLE
regression about tab's ink-bar width after upgrade to 1.1.3

### ISSUE THREAD

**Issue by bvahdat** [angular/material#10386]
**Actual Behavior**:
 -  `What is the issue?` The tab component's ink bar becomes sometimes either too long or too short on iOS, both Safari & Chrome. That's, it becomes either shorter than the active tab's width or crosses the border to the next tab. If I locally revert the made changes through #10136 then the issue is not given anymore!
 -  `What is the expected behavior?` The ink bar should be as wide as the active tab's width.

**CodePen** (or steps to reproduce the issue): 
 -  `CodePen Demo which shows your issue:` I'm reusing the same CodePen being provided by #10136, that's http://codepen.io/shortfuse/pen/WoYQdW
Please note that the behaviour is not deterministic and you may need to reload the CodePen many times on your device to reproduce the issue.
 -  `Details:` Please see the following two screenshots:

![img_2458](https://cloud.githubusercontent.com/assets/3122177/22757131/ef705796-ee49-11e6-94f3-f898f01ef9ff.jpg)

![img_2457](https://cloud.githubusercontent.com/assets/3122177/22757122/e8beedb8-ee49-11e6-8fed-e4d932a23a18.jpg)

**Angular Versions**: *
 -  `Angular Version:` 1.6.1
 -  `Angular Material Version:` 1.1.3

**Additional Information**:
 -  `Browser Type:` Safari & Chrome
 -  `Browser Version:` Safari on iOS 10.2.1 & Chrome Version 56.0.2924.79
 -  `OS:` iOS 10.2.1
 -  `Stack Traces:` N/A

----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.

**Comment by clshortfuse** [comment]
On iOS, even with the Chrome app, the rendering engine is actual Safari Mobile. I can only reproduce something similar on the Chrome rendering engine while resizing the window, but it fixes itself after it's done resizing. 

My suspicion is https://github.com/angular/material/blob/5b799e214233b42625359c9c663499d273cc40fb/src/components/tabs/js/tabsController.js#L822 is not being called sometimes on Safari Mobile

**Comment by bvahdat** [comment]
@clshortfuse you've provided the following CodePen link by your fix for #10136

>http://codepen.io/shortfuse/pen/WoYQdW (master with overriding CSS rule)

However that _doesn't_ include your fix as the JavaScript Setting of that CodePen points to the latest commit being 491d139 which is this:

https://cdn.gitcdn.link/cdn/angular/bower-material/v1.1.1-master-491d139/angular-material.js

That still contains a call to `updatePagingWidth()` function through `updatePagination()` which you've removed by your pushed commit 5b799e2.

To me that means you've tested your fix with the wrong codebase version on CodePen, or am I wrong?

cc: @ErinCoughlan @DevVersion

**Comment by clshortfuse** [comment]
@bvahdat According to the PR comment, I did a localhost test with http://codepen.io/shortfuse/pen/LbXVWP likely with Browserstack.

The note of 

>Please note that the behaviour is not deterministic and you may need to reload the CodePen many times on your device to reproduce the issue.

Lead me to believe that the issue could be a timing issue related to how the window is sized. Tinkering with timeouts or forcing ticks could provide a suitable workaround.

**Comment by bvahdat** [comment]
@clshortfuse thanks for your reply 👍 

Always when the ink bar has not the correct length (it's almost always longer than shorter) then after starting to scroll, the tab length adjusts itself and becomes of the correct width.

**Comment by bvahdat** [comment]
@clshortfuse I managed to resolve my issue on iOS Safari/Chrome which was a side effect of using the `md-center-tabs` directive. It's documentation says:

>When enabled, tabs will be centered provided there is no need for pagination

In my case pagination was required. On desktop & tablet the behaviour was as expected but apparently on mobile it caused the side effect mentioned by this issue. For desktop & tablet I then styled the tabs positioning by myself through the `md-pagination-wrapper` element.

Thanks for your support 👍
